lib/chai/core/assertions.js

```diff
@@ -587,6 +606,7 @@ module.exports = function (chai, _) {
    * @api public
    */
 
+  // Should this also be updated?
```

Comment:
Originally we discussed only isAbove and isBelow methods to support dates, but these seem connected as well.